src/styles/design-token.json

```diff
@@ -1329,6 +1329,36 @@
                     }
                 }
             }
+        },
+        "boxShadow": {
```

Comment:
Is this from auto-generated file?

Reply:
No @balakrishna-binary , boxShadow was added by me as per the specification mentioned in elevation section in ZH